### PR TITLE
Recreate objects after deletion fails

### PR DESCRIPTION
**Pull request by Natim**
kinto.py is struggeling recreating collection, buckets, groups and records with the ``safe=True`` flag if they have been deleted.

These are the test cases that are covering the matter.

### CODE REVIEW

**Comment by leplatrem** [comment]
I could fix the problem easily :)

File: kinto/tests/test_views_records.py [hunk 1]
```diff
@@ -227,3 +227,12 @@ def test_records_should_accept_client_accept(self):
                      MINIMALIST_RECORD,
                      headers=headers,
                      status=200)
+
+    def test_records_can_be_created_after_deletion(self):
+        self.app.delete(self.record_url,
+                        headers=self.headers,
+                        status=200)
```
Comment (leplatrem) [inline kinto/tests/test_views_records.py]: nit: 200 is superfluous

File: kinto/tests/test_views_groups.py [hunk 2]
```diff
@@ -158,6 +158,13 @@ def test_group_member_removal_updates_user_principals_with_patch(self):
         self.assertEquals(self.permission.user_principals('mat'),
                           {group_url})
 
+    def test_groups_can_be_created_after_deletion(self):
+        self.app.delete(self.group_url, headers=self.headers)
```
Comment (leplatrem) [inline kinto/tests/test_views_groups.py]: The group is not created in setup, so you should create it before deleting it.

**Comment by Natim** [comment]
The problem have been merged into Cliquet master.

**Comment by Natim** [comment]
Tests are passing with cliquet-master. We can/should wait for a cliquet release before merging.

**Comment by almet** [comment]
Refs https://github.com/mozilla-services/cliquet/commit/2c2f0337923272a52caa3a0e32a1bfd23f8e374b

**Comment by Natim** [comment]
cliquet 3.1.2 have been released with this fix.

**Comment by Natim** [comment]
Tests are now green :+1:

**Pull request merged**